### ISSUE TITLE
1.20.1 Tebex ERROR

### ISSUE THREAD

**Issue by GeoGb1** [tebexio/Tebex-Minecraft#6]
When the server starts, i get these errors and every tebex command is broken, even /buy is an empty gui.
```
[23:18:48 WARN]: [Tebex] Failed to get online commands: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]: java.util.concurrent.CompletionException: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:649)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.request.TebexRequest$1.onResponse(TebexRequest.java:120)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[23:18:48 WARN]:        at java.base/java.lang.Thread.run(Thread.java:840)
[23:18:48 WARN]: Caused by: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.gson.JsonElement.getAsInt(JsonElement.java:239)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.SDK.lambda$getOnlineCommands$3(SDK.java:213)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646)
[23:18:48 WARN]:        ... 7 more
[23:18:48 WARN]: [Tebex] Failed to get online commands: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]: java.util.concurrent.CompletionException: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:649)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.request.TebexRequest$1.onResponse(TebexRequest.java:120)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[23:18:48 WARN]:        at java.base/java.lang.Thread.run(Thread.java:840)
[23:18:48 WARN]: Caused by: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.gson.JsonElement.getAsInt(JsonElement.java:239)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.SDK.lambda$getOnlineCommands$3(SDK.java:213)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646)
[23:18:48 WARN]:        ... 7 more
[23:18:48 WARN]: [Tebex] Failed to get online commands: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]: java.util.concurrent.CompletionException: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:649)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.request.TebexRequest$1.onResponse(TebexRequest.java:120)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[23:18:48 WARN]:        at java.base/java.lang.Thread.run(Thread.java:840)
[23:18:48 WARN]: Caused by: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.gson.JsonElement.getAsInt(JsonElement.java:239)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.SDK.lambda$getOnlineCommands$3(SDK.java:213)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646)
[23:18:48 WARN]:        ... 7 more
[23:18:48 WARN]: [Tebex] Failed to get online commands: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]: java.util.concurrent.CompletionException: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:649)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.request.TebexRequest$1.onResponse(TebexRequest.java:120)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[23:18:48 WARN]:        at java.base/java.lang.Thread.run(Thread.java:840)
[23:18:48 WARN]: Caused by: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.gson.JsonElement.getAsInt(JsonElement.java:239)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.SDK.lambda$getOnlineCommands$3(SDK.java:213)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646)
[23:18:48 WARN]:        ... 7 more
[23:18:48 WARN]: [Tebex] Failed to get online commands: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]: java.util.concurrent.CompletionException: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:649)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.request.TebexRequest$1.onResponse(TebexRequest.java:120)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[23:18:48 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[23:18:48 WARN]:        at java.base/java.lang.Thread.run(Thread.java:840)
[23:18:48 WARN]: Caused by: java.lang.UnsupportedOperationException: JsonNull
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.plugin.libs.gson.JsonElement.getAsInt(JsonElement.java:239)
[23:18:48 WARN]:        at tebex-bukkit-2.0.0.jar//io.tebex.sdk.SDK.lambda$getOnlineCommands$3(SDK.java:213)
[23:18:48 WARN]:        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646)
[23:18:48 WARN]:        ... 7 more
```

**Comment by WildBamaBoy** [comment]
Make sure that your secret key is appropriately set. If you've migrated from a previous version there may be issues reading and setting your secret key automatically, which would cause this issue.

Use `/tebex secret <your-key-here>` to set the key, or set directly in the plugin's configuration file. Afterwards restart the server and let us know if the issue continues.

**Comment by WildBamaBoy** [comment]
This is fixed in the latest release 2.0.1
